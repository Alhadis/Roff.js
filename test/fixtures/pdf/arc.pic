.PS
	spline right then up then left then down ->
	move right
	move right
	spline left then up right then down right ->;
.PE
