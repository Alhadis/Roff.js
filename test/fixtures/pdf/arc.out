x T pdf
x res 72000 1 1
x init
x F fixtures/arc.pic
p1
V12000
H72000
DFd
h108000
n12000 0
V12000
H72000
n12000 0
V12000
H72000
DFg 0
v31536
md
s10000
DP -1800 -7200 3600 0
n12000 0
V12000
H72000
x font 5 TR
f5
cA
Dt 8000 0
cZ
H72000
n12000 0
V43536
H72000
Dp -1800 -7200 3600 0
n12000 0
V12000
H72000
Dt -1000 0
h1000
n12000 0
V43536
H72000
D~ 36000 0 0 -36000 -36000 0 0 35280
n12000 0
V43536
H180000
DP -6336 -3816 2520 -2520
n12000 0
V12000
H72000
Dt -1000 0
H72000
n12000 0
V43536
H180000
Dp -6336 -3816 2520 -2520
n12000 0
V12000
H72000
Dt -1000 0
h1000
n12000 0
V43536
H144000
D~ -36000 0 36000 -36000 35496 35496
n12000 0
x trailer
V792000
x stop
